en/docs/fhir/guides/metadata-api.md

```diff
@@ -0,0 +1,290 @@
+# FHIR R4 Metadata Service
+
+## Introduction
+This service provides implementation of FHIR Metadata API. This implements 
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+!!! note
+        Supported FHIR version is 4.0.1.
+
```

Comment:
Let's add the note of implementation flavours

{!includes/bal-mi-note.md!}

Reply:
Added